dozer-types/src/arrow_types/from_arrow.rs

```diff
@@ -175,7 +175,7 @@ pub fn map_arrow_to_dozer_type(dt: &DataType) -> Result<FieldType, FromArrowErro
         // DataType::Dictionary(_, _) => {}
         // DataType::Decimal128(_, _) => {}
         // DataType::Decimal256(_, _) => {}
-        // DataType::Map(_, _) => {}
+        DataType::Map(_, _) => Ok(FieldType::Json),
```

Comment:
If we don't implement conversion in `map_value_to_dozer_field`, we shouldn't do here.